data_structures/src/superblock.rs

```diff
@@ -182,6 +183,23 @@ impl SuperBlockState {
             }
         }
     }
+
+    /// Produces a `SuperBlock` that includes the blocks in `block_headers` if there is at least one of them.
+    /// `ars_pkh_keys` will be used to validate all the superblock votes received for the
+    /// next superblock. The votes for the current superblock must be validated using
+    /// `previous_ars_identities`. The ordered bn256 keys will be merkelized and appended to the superblock
```

Comment:
I think the description hast to be updated

Reply:
very true haha